HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+                },
+                "EthernetInterfaces": {},
+                "LogServices": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "ManagerForChassis": {
+                            "ReadRequirement": "Supported"
+                        }
+                    }
+                }
+            },
+            "ActionRequirements": {
+                "Reset": {
+                    "Parameters": {
+                        "ResetType": {
+                            "ParameterValues": [
+                                "ForceRestart"
+                            ]
+                        }
+                    }
+                }
+            }
+        },
+        "ManagerAccount": {
+            "PropertyRequirements": {
+                "AccountTypes": {},
+                "Enabled": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "Role": {}
+                    }
+                },
+                "Password": {},
+                "PasswordChangeRequired": {},
+                "PasswordExpiration": {},
+                "RoleId": {},
+                "UserName": {}
+            }
+        },
+        "ManagerNetworkProtocol": {
+            "PropertyRequirements": {
+                "HostName": {},
+                "FQDN": {},
+                "HTTP": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "HTTPS": {
+                    "PropertyRequirements": {
+                        "Certificates": {},
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "NTP": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "NetworkSuppliedServers": {},
+                        "NTPServers": {},
+                        "Port": {},
+                        "ProtocolEnabled": {}
+                    }
+                },
+                "SSH": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {}
+                    }
+                },
+                "SSDP": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "ProtocolEnabled": {},
+                        "Port": {},
+                        "NotifyMulticastIntervalSeconds": {},
+                        "NotifyIPv6Scope": {},
+                        "NotifyTTL": {}
+                    }
+                }
+            }
+        },
+        "OutboundConnection": {
```

Comment:
Would need OutboundConnectionCollection to show insert/delete requirements. Might also need to change "Recommended" to "IfImplemented", or something else to cover the case where the service supports the capability, but the collection is empty due to no one configuring it yet.

Reply:
Adding the Collection insert/delete requirements.  I think "Recommended" at the resource level should do what we want.  If the collection is non-empty, the properties listed become mandatory.  But if collection is empty, the resource-level is only recommended so none of those property-level requirements get tested (or enforced).

Reply:
Scratch that; since outbound connections are not mandatory, leave as Recommended. If we ever elevate them to be mandatory, then this needs to be IfPopulated.